### ISSUE TITLE
Exemplet fungerar inte

### ISSUE THREAD

**Issue by bappobrad69** [Blatzar/schoolsoft-api-app#5]
```
import schoolsoft_api
name, password, school = 'name', 'password', 'school' # Place real values here.
app_key = schoolsoft_api.get_app_key(name, password, school)
org_id = app_key['orgs'][0]['orgId']
token = schoolsoft_api.get_updated_token(school, app_key_json=app_key)['token']
schoolsoft_api.get_lunch(token, school, org_id)

```

Jag ändrade självklart name, password och school till de rätta men får detta error:
```
org_id = app_key['orgs'][0]['orgId']
KeyError: 'orgs'
```

**Comment by Blatzar** [comment]
Schoolsoft var nere igår eller förrgår tror jag, kolla om exemplet fungerar nu, det gör för mig. Säkert bara otur när du testade

**Comment by bappobrad69** [comment]
Har fortfarande samma problem, requesten till schoolsofts rest api resulterar också i ett 403

**Comment by Blatzar** [comment]
Kan du logga in på mobilappen?

Kolla också [skollistan](https://sms.schoolsoft.se/rest/app/schoollist/prod) efter studentLoginMethods, parentLoginMethods eller teacherLoginMethods beroende på vilket slags konto du försöker nå från och rapportera resultatet.

Det är bara testat som student login och använder logintype '4' så jag antar att apin inte skulle funka för exempelvis 'Pysslingen Förskolor' eftersom de bara har loginMethods '3'. Vissa skolor använder sitt eget inloggningsssystem eller bankID vilket inte stöds än.

'usertype' hardcodat i apin, det kanske är problemet, loggar du in som elev, lärare eller förälder?

**Comment by bappobrad69** [comment]
Ok tack så mycket, min skola använder sig av logintype 3 vilket gör att min request går igenom. Men nu har jag ett nytt problem.

```
response = requests.post('https://sms.schoolsoft.se/pysslingen/rest/app/login', data={
    'identification': 'namn',
    'verification': 'lösenord',
    'logintype': '3',
    'usertype': '1'
})
```
Detta ger mig en status på 200 men **response.json()** ger mig ett tomt object. Det jag antar är att apiet inte stödjs då jag tror inloggningen hos mig hanteras av academedia.

**Comment by Blatzar** [comment]
> Det jag antar är att apiet inte stödjs då jag tror inloggningen hos mig hanteras av academedia.

Ja, får väl lägga till detta till readme. Du får gärna lägga upp apin för academedia för schoolsoft om du får tag på den.
